.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Integration Tests
```

Comment:
if this isn't working, I'd just remove it for now and we can revisit with the new infra

Reply:
099139e85516ae696ad68e53cae661f121addb5c